src/test/java/core/basesyntax/service/impl/FruitTransactionParserServiceImplTest.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitReport;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.FruitTransactionParserService;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FruitTransactionParserServiceImplTest {
+    private FruitTransactionParserService fruitTransactionParserService =
+            new FruitTransactionParserServiceImpl();
+
+    @Before
+    public void cleanBefore() {
+        Storage.fruits.clear();
```

Comment:
>clear before
>test
>clear after
>clear before
....


in the method before better to add some common logic for every single tests (as an example: add data to storage, set up model classes, etc.) 
in method after - clear data from storage 
Fix it in all places